src/Service/PaymentMethodsFilterService.php

```diff
@@ -249,6 +209,7 @@ public function getAvailableNonGiftcardsPaymentMethods(
 
     public function filterAdyenPaymentMethodsByType(array $paymentMethods, string $type): array
     {
+        //Filter for if giftcard PM is disabled do not allow any giftcard mechanism.
```

Comment:
Is this a `TODO` item or a valid comment?

Reply:
This is already handled in PaymentSubscriber(line 232-244)